vignettes/censusviz_vignette.Rmd

```diff
@@ -1,36 +1,109 @@
 ---
-title: "censusviz_vignette"
-author: "Rose Porta"
-date: "4/2/2022"
-output: html_document
+title: "censusviz"
+author: "Irene Foster, Catherine Park, Rose Porta"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{censusviz}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
 ---
 
-```{r setup, include=FALSE}
-knitr::opts_chunk$set(echo = TRUE)
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
 ```
 
+```{r setup}
+library(censusviz)
+library(ggplot2)
+library(dplyr)
+```
+
+## About censusviz
+
+The [`censusviz`](https://github.com/rporta23/censusviz) package allows users to explore and visualize historical racial demographic census data for any region in the United States using leaflet maps integrated into a shiny app. This will help facilitate with the process of using historical census data for analyzing and visualizing. 
+
+This package was inspired by the `nepm` package. 
+
+## Getting Start
+
+In the example that will follow, we will be using New York, Madison county to illustrate our package's functionality. 
+
+### Reading in Data
+
+The three data frames used in this package are [`tracts_long_all.rda`](https://github.com/CJParkNW/cenviz_data), [`tracts_demo_joined1.rda`](https://github.com/rporta23/censusviz-data), and [`tracts_demo_joined2.rda`](https://github.com/CJParkNW/cenviz_data). Each one is read in directly from a GitHub repository due to their large sizes. `tracts_demo_joined1.rda` and `tracts_demo_joined2.rda` are combined when calling function get_data_wide() to create data_wide. `tracts_long_all.rda` will be called with get_data_long().
+
 ```{r}
 data_wide <- get_data_wide()
+
+data_long <- get_data_long()
 ```
 
+### Filtering Data by State, County
+
+The user can call filter_data_wide() and filter_data_long() with the proper state and county to filter out only the data that is related to that area. Here we are filtering for data only related to the county Madison in New York.
+
 ```{r}
 data_wide_filtered <- filter_data_wide(data_wide, "New York", "Madison")
+
+data_long_filtered <- filter_data_long(data_long, "New York", "Madison")
 ```
 
+### Creating Sampling Dots
+
+After filtering the data, the user can use data_wide_filtered to call create_dots() to produce the sampling dots for each population to visualize on a leaflet map. 
+
```

Comment:
I think it might make sense to explain that create_dots() samples random points within each census tract for each racial category such that each dot represents num_people people (100 by default) in order to show the distribution of racial demographics across the county. I think it may be a bit confusing to the user to just say that it "produces the sampling dots"